src/App.jsx

```diff
@@ -1,50 +1,25 @@
-import { useState, useEffect } from 'react';
-import { getArticles } from './api';
+import { Routes, Route, Link, Navigate } from 'react-router-dom';
+import { ArticleList } from './components/articles/ArticleList';
+import { Article } from './components/articles/Article';
 import './App.css';
 
 function App() {
   return (
     <main>
       <header>
-        <h1>NC News</h1>
+        <Link to={'/'}>
+          <h1>NC News</h1>
+        </Link>
       </header>
       <nav>Nav</nav>
-      <ArticleList />
+      <Routes>
+        <Route path="/" element={<ArticleList />} />
+      </Routes>
+      <Routes>
+        <Route path="/articles/:article_id" element={<Article />} />
+      </Routes>
```

Comment:
Got `No routes matched location "/articles/<article_id>"` warning. put line 19 on line 16.5 should fix that. No need to put them into different Routes tags